scripts/package.ps1

```diff
@@ -45,7 +67,11 @@ wget http://download.sysinternals.com/files/SDelete.zip -OutFile sdelete.zip
 ./sdelete.exe /accepteula -z c:
 
 mkdir C:\Windows\Panther\Unattend
-copy-item a:\postunattend.xml C:\Windows\Panther\Unattend\unattend.xml
+if ($OS.Version -eq "6.1.7601") {
```

Comment:
would like to see this follow some naming convention instead of having to switch by version